build.sh

```diff
@@ -0,0 +1,5 @@
+export PKG_CONFIG=/Users/sahaschitlange/go/bin/pkg-config
```

Comment:
This looks like a local script to build flux. Can you remove this? I think you can do `make build` to do the same thing.

Reply:
Done!